layouts/section/job-posting.fr.html

```diff
@@ -1,144 +1,151 @@
+    <form enctype="multipart/form-data" class="text-left" role="form" id="contactForm">
+      <input type="hidden" id="jobId" name="jobId" value="{{.Params.leverId}}" />
+      <!-- Full Name -->
+      <div class="form-group">
+        <label for="name">Nom complet</label>
+        <input type="text" class="form-control validate-required" id="name" name="name" aria-required="true"
+          aria-describedby="name-required" />
+        <span id="name-required" class="error-message validate-required-message" style="display: none;">Ce champ est
+          obligatoire</span>
+      </div>
+      <!-- Email -->
+      <div class="form-group">
+        <label for="email">Courriel</label>
+        <input type="email" name="email" class="form-control validate-required validate-email" id="email"
+          aria-required="true" aria-describedby="email-required" />
+        <span id="email-required" class="error-message validate-required-message" style="display: none;">Ce champ est
+          obligatoire</span>
+      </div>
+
+      <!-- Resume -->
+
+      <label for="resume">CV</label>
+      <div class="form-group">
+        <!-- <label for="resume">Resume</label> -->
+        <div class="browse">
+          <label>Parcourir...
+            <input class="resume" type="file" id="resume" name="resume"
+              accept="application/msword, application/pdf, text/plain" required style="display: none;" />
+          </label>
+
+        </div>
+        <span class="no-file-chosen" id="no-file-chosen-text">Aucun fichier sélectionné</span>
+
+
+      </div>
+      <!-- Body -->
+
+      <div class="form-group">
+        <label for="body">Dites-nous, en 300 mots ou moins, pourquoi vous voulez travailler avec nous et, le cas
+          échéant, ce que l’on devrait savoir au sujet de vos échantillons de travail.</label>
+        <textarea class="form-control validate-required validate-maxwords" id="body" name="body" rows="5"
+          aria-required="true" aria-describedby="body-required word-count-message"></textarea>
+        <span id="body-required" class="error-message validate-required-message" style="display: none;">Ce champ est
+          obligatoire<br />
+        </span>
+        <span id="word-count-message" class="help-text">
+          <span id="word-count">300</span> mots restants</span>
+
+      </div>
+      <!-- Samples of your work -->
+      <div class="form-group">
+        <label for="samples">Échantillons de votre travail<span class="alt-label"> (Facultatif)</span></label>
+        <div>
+          <span class="url">Liens URL SVP!</span>
+
+        </div>
+        <textarea class="form-control validate-required validate-maxwords" id="samples" name="samples" rows="5"
+          aria-required="true" aria-describedby="body-required word-count-message"></textarea>
+      </div>
+      <!-- Submit-->
+      <div>
+        <button type="submit" class="submit-btn">
+          Soumettre<img src="/img/icons/arrow-right-white.svg" />
```

Comment:
If you want to not add alt text then you need to add either `role="none"` or `role="presentation"` to this element. 

I would go with presentation: 
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_presentation_role